src/main/resources/i18n/messages_en_US.properties

```diff
@@ -114,6 +114,7 @@ Important Links:<br/>\
 </ol>
 
 #### Attack Vector Description
+UNRESTRICTED_FILE_UPLOAD_UNCONTROLLED_RESOURCE_CONSUPTION=Maximum uploaded file size is not limited.
```

Comment:
please add this to other locale files as well.

Reply:
I can add placeholders, but I don't speak the other languages, so can't add actual translations. I could do google translate, if that is acceptable.

Reply:
add just in message.properties